RSignatureVerifier/Fetcher.swift

```diff
@@ -0,0 +1,33 @@
+internal class Fetcher {
```

Comment:
Can it be a final class?

Reply:
It can but is there any performance gain in this case?

Reply:
Can't be final because of Mocks

Reply:
Yes there is a gain using final. I could not retrieve @donnie-jp 's article but this is a blog post from Apple about final class:
https://developer.apple.com/swift/blog/?id=27
If you create a FetcherProtocol, you won't need inheritance for your Mock.